terraform-infra-approvals/fis-hmc-api.json

```diff
@@ -0,0 +1,8 @@
+{
+  "resources": [
+    {"type": "azurerm_servicebus_subscription_rule"}
+  ],
+  "module_calls": [
+    {"source":  "git@github.com:hmcts/terraform-module-servicebus-subscription?ref=require_session"}
```

Comment:
why? you should be able to test this without allowlisting it, non master branches are generally not needed unless needed for a transition

Reply:
> why? you should be able to test this without allowlisting it, non master branches are generally not needed unless needed for a transition

Hi timja, I have tried in DEMO environment. Jenkins build fails saying the resource need to be whitelisted. Reference : https://build.platform.hmcts.net/blue/organizations/jenkins/HMCTS_d_to_i%2Ffis-hmc-api/detail/demo/2/pipeline/53/.
Kindly could you please check this ?